Makefile

```diff
@@ -38,7 +38,7 @@ artifacts/honeycomb-metrics-config.yaml: config-generator.jq vendor-fixtures/hos
 
 # copy hostmetrics metadata yaml file from the OpenTelemetry Collector repository, and prepend a note saying it's vendored
 vendor-fixtures/hostmetrics-receiver-metadata.yaml:
-	REMOTE_PATH='https://raw.githubusercontent.com/open-telemetry/opentelemetry-collector/main/receiver/hostmetricsreceiver/metadata.yaml'; \
+	REMOTE_PATH='https://raw.githubusercontent.com/open-telemetry/opentelemetry-collector-contrib/141da3a5c4a1bf1570372e2890af383dd833167b/receiver/hostmetricsreceiver/metadata.yaml'; \
```

Comment:
This change required because the file `hostmetricsreceiver/metadata.yaml` no longer exists in the latest version of the repo. We have to link to specific version where it existed.